tests/test_examples.py

```diff
@@ -49,7 +49,7 @@ def _rmtree(directory):
 def run_pip(args):
     environ = os.environ.copy()
     environ['PYTHONPATH'] = ':'.join(sys.path)
-    subprocess.call([sys.executable, '-m', 'pip'] + args + ["--use-feature=in-tree-build", "."],
+    subprocess.call([sys.executable, '-m', 'pip'] + args + ["."],
```

Comment:
Seems this fails on Fedora 35, probably older pip?  Would you mind adding
a simple if-else condition per PIP version?
https://copr.fedorainfracloud.org/coprs/praiskup/argparse-manpage-pull-requests/build/4642267/